### ISSUE TITLE
Exception with randomize_ligand: yes

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#654]
Just ran into this
```
Exception encountered when processing molecule MOL.
[ERROR] Traceback (most recent call last):
  File "/Users/choderaj/github/openeye/openmm_orion/YankCubes/cubes.py", line 445, in process
    yaml_builder.build_experiments()
  File "/Users/choderaj/miniconda3/envs/orion/lib/python3.5/site-packages/yank/yamlbuild.py", line 1333, in build_experiments
    self._run_experiment(combination, output_dir)
  File "/Users/choderaj/miniconda3/envs/orion/lib/python3.5/site-packages/yank/yamlbuild.py", line 2519, in _run_experiment
    yank.create(thermodynamic_state, *phases, restraint_type=restraint_type)
  File "/Users/choderaj/miniconda3/envs/orion/lib/python3.5/site-packages/yank/yank.py", line 288, in create
    self._create_phase(thermodynamic_state, phase, restraint_type)
  File "/Users/choderaj/miniconda3/envs/orion/lib/python3.5/site-packages/yank/yank.py", line 514, in _create_phase
    self._randomize_ligand_sigma_multiplier * restraints.getReceptorRadiusOfGyration(),
AttributeError: 'Harmonic' object has no attribute 'getReceptorRadiusOfGyration'
```

**Comment by Lnaden** [comment]
Looks like that [function was deleted](https://github.com/choderalab/yank/commit/70d121b4dab3d6a9b335fcea6168f9c2f4c8ece6#diff-88a9ac403b98991ed235fb182192e277L333) when you started the refactor of the restraints code. I guess we just have not used the `randomize_ligand` function since then. I can add that function back in.

**Comment by andrrizzi** [comment]
The function to compute the radius of gyration is currently unused in `restraints`. I'm thinking about moving it as a module function in `pipeline` where there will be utility functions to handle positions (e.g. compute distances, pull close ligand and receptor). Any objections?

I'll correct the method call in the next few days while refactoring `yank`.

**Comment by jchodera** [comment]
That's odd! The radius of gyration should be used to compute the spring constant when setting up the harmonic restraints if no lengthscale is defined. It should certainly go somewhere, and it does seem to make sense that it could be computed in the `pipeline` if that has access to the configurations of receptor and ligand.

**Comment by andrrizzi** [comment]
You're right! The code was actually duplicated in the function that determines the spring constant for the `Harmonic` restraint, that's how it ended up not being used. I'll remove the code duplication.

**Comment by andrrizzi** [comment]
This was fixed in #662 .
